src/main/resources/application-postgres.yaml

```diff
@@ -0,0 +1,8 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/visit_scheduler_db
+    username: admin
+    password: admin_password
+  flyway:
+    enabled: true
```

Comment:
looks like this is true anyway, so this might confuse people having it here as well